Assets/Runtime/UIFrame.cs

```diff
@@ -119,6 +119,24 @@ public class UIFrame : MonoBehaviour
             windowLayer.HideScreenById(screenId, animate);
         }
         
+        /// <summary>
+        /// Closes the Window with the given Id.
```

Comment:
Don't need this comment.

Reply:
all other class member had documentation with such quality
let keep it align otherwise it will be even worse